config/environments/production.rb

```diff
@@ -28,7 +32,7 @@
   # config.assets.css_compressor = :sass
 
   # Do not fallback to assets pipeline if a precompiled asset is missed.
-  config.assets.compile = false
+  config.assets.compile = true
```

Comment:
we shouldn't have this on Production, it will make the app super slowdown, any reason we need to turn this on @htoo-eain-lwin

Reply:
Hello @andyduong1920, I am working on tests failing #8. 
For enabling `assets.compile: true` on production, I noticed that is not the usual way and mentioned in this PR description.  I already created the new [issue](https://github.com/htoo-eain-lwin/ruby-crawler/issues/6 ) before creating this PR. 
I will tackle after working on test. Thanks for your insights Andy.

Reply:
Noted @htoo-eain-lwin

Reply:
I solved this and close the issue #6.